python/tests/test_testclient_shared_connection.py

```
@@ -0,0 +1,356 @@
+def test_the_escape_hatch_serves_a_worker_that_iterates():
+    """The fix the error names must work for the case that raises it.
+
+    This is the example in the testing guide: a thread of the test iterates a
+    QuerySet, which holds a cursor open, while the client serves requests.
+    """
+    for i in range(20):
+        User.objects.create(username=f"u{i}", email=f"u{i}@example.com")
+
+    exported: list[str] = []
+
+    def export():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `export`

Add return type annotation: `None`

(ANN202)